.travis.yml

```diff
@@ -66,9 +66,6 @@ cache:
 env:
         - CONF=""
         - CONF="--disable-abi-compat"
-        - CONF="--enable-schedule-sp"
-        - CONF="--enable-schedule-iquery"
-        - CONF="--enable-schedule-scalable"
         - CONF="--enable-dpdk-zero-copy"
```

Comment:
What do these changes have to do with the subject of this PR? Changes to Travis should at minimum be a separate commit (these are not "linux-generic: buffer" changes) and should probably be a separate PR with its own rationale.

Reply:
Agree that. Travis changes should be in a separate commit.